con-base/src/main/java/ba/codecentric/base/domain/Facility.java

```diff
@@ -0,0 +1,103 @@
+package ba.codecentric.base.domain;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Size;
+
+@Entity
+public class Facility {
+
+    @Id
+    @GeneratedValue
+    private Integer id;
+
+    @Column(name = "facility_name")
+    @NotBlank(message = "Please enter name of the facility")
+    @Size(max = 80, message = "Facility name is too long, 80 characters allowed")
+    private String facilityName;
+
+    @NotBlank(message = "Please add facility description")
+    @Size(max = 400, message = "Description is too long, 400 characters allowed")
+    private String description;
+
+    @NotBlank(message = "Please add facility image")
+    @Size(max = 128, message = "Address of image is too long, 128 characters allowed")
+    private String image;
+
```

Comment:
Here we have validation but there is nothing in controller. So it gets validated upon saving which will throw exception. Anyway size of 128 is too small for image. And if it is in fact url of image and not image itself better naming would be imageUrl.